docker/zenml-server-hf-spaces.Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM zenmldocker/zenml-server:latest
+
+ENV ZENML_ANALYTICS_OPT_IN=true
+ENV ENV_ZENML_SERVER_DEPLOYMENT_TYPE="hf_spaces"
```

Comment:
This ENV variable is not being picked up by Mixpanel. Not sure if there's a better way to handle making sure analytics are turned on for this deployment type. @bcdurak for when you're back?